libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1133,6 +1133,13 @@ std::shared_ptr<WaveTrack> WaveTrackFactory::Create(sampleFormat format, double
 }
 
 TrackListHolder WaveTrackFactory::Create(size_t nChannels)
+{
+   assert(nChannels > 0);
+   assert(nChannels <= 2);
+   return Create(nChannels, QualitySettings::SampleFormatChoice(), mRate.GetRate());
+}
+
+TrackListHolder WaveTrackFactory::CreateMany(size_t nChannels)
 {
    return Create(nChannels, QualitySettings::SampleFormatChoice(), mRate.GetRate());
```

Comment:
Shouldn't this be calling `CreateMany` ?

Reply:
Wait to read a later commit that makes the distinction in return types.